static/app/utils/useTeamsById.spec.tsx

```diff
@@ -0,0 +1,133 @@
+import {reactHooks} from 'sentry-test/reactTestingLibrary';
+
+import OrganizationStore from 'sentry/stores/organizationStore';
+import TeamStore from 'sentry/stores/teamStore';
+import {QueryClient, QueryClientProvider} from 'sentry/utils/queryClient';
+
+import {useTeamsById as useTeamsById} from './useTeamsById';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+});
+
+describe('useTeamsV2', function () {
+  const org = TestStubs.Organization();
```

Comment:
low-level of care, but gentle nudge to import from `sentry-fixtures` if possible

but i know habits need time to change, it's not the end of the world if there's a few more TestStubs out there.